assemblies/rosa-troubleshoot-cli.adoc

```diff
@@ -0,0 +1,14 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-troubleshooting-cli"]
+
+= Troubleshooting with `rosa`
```

Comment:
Per recent discussion, no formatting in titles

Reply:
Fixed.